CONTRIBUTING.md

```diff
@@ -231,6 +231,32 @@ The following guidelines must be adhered to if you are writing code to be merged
   * We only need one extensive set of documentation rather than individual
     sets scoped per Slurm charm.
 
+#### Adding new internal packages
+
+If you want to add a new internal Python package to the monorepo, you can run:
+
+```bash
+uv init --lib packages/pkg-name
```

Comment:
Since we updated the directory name to `pkgs`, we should also update the name here as well:

```suggestion
uv init --lib pkgs/pkg-name
```